.github/workflows/docs.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build and Deploy Documentation
+on:
+  push:
+    branches:
+      - develop
+      - feature/*
+      - main/*
+      - bugfix/*
+      - release/*
+    paths:
+      - docs/**
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
Not sure if all these should trigger the action. Shouldn't it only be on pushes to develop?

Reply:
If I understand this correctly this should create a documentation build for all the branches that have sphinx documentation and get updated. Where are the documentation files uploaded though?

Reply:
L38

Reply:
the uploads are as artifacts on github actions.  That is not RTD.  The artifacts will allow you to download the whole documentation.

To answer @WalterKolczynski-NOAA, I believe we want to build the latest documentation (`develop`), but we also want to build documentation for releases for sure.  Not sure about bugfixes and feature branches.

I know RTD does allow one to toggle between documentations between releases and latest.

Reply:
Okay, if RTD can handle branches, I'm satisfied.